src/assertions.ts

```diff
@@ -32,12 +32,17 @@ let inAttributes = false;
  */
 let inSkip = false;
 
+/**
+ * Keeps track of the current patch context.
+ */
+let patchContext: {}|null = null;
```

Comment:
Isn't this context.ts?

Reply:
Changed this to just track the `inPatch` state, since we don't really care about the context itself.